src/indexers/hosted/indexer.ts

```diff
@@ -0,0 +1,151 @@
+import { CortexApiClient } from "../../api-client";
+
+export type IndexerExecutionStatus = "success" | "failure" | "inProgress";
+
+export type IndexerExecutionResult = {
+  status: IndexerExecutionStatus;
+  startTimeUtc: string;
+  endTimeUtc?: string;
+  errors: string[];
+  warnings: string[];
+};
+
+export type IndexerExecutionHistory = {
+  results: IndexerExecutionResult[]; // the size of this array is bounded - see trimIndexerExecutionHistoryIfNeeded
```

Comment:
What is `trimIndexerExecutionHistoryIfNeeded`?

Reply:
Cut and paste 🤦 will remove